pages/common/git-show.md

```diff
@@ -7,13 +7,9 @@
 
 `git show`
 
-- Show information about a given commit:
+- Show information about a given commit | tag | branch:
```

Comment:
```suggestion
- Show information about a given commit/tag/branch:
```
In writing this is probably better but I want to hear your opinion too

Reply:
True. I still think that it should be involved in the wording that it refers to the tip of the branch and not the whole branch but I'm unable to come up with the words to express it